add-thread-count-fields (#42)
elastic/enterprise-search-sharepoint-server-connector: README.md

retry_count: 3
```

#### `sharepoint_sync_thread_count`

> The number of threads the connector will run parallelly for fetching documents from the SharePoint server. By default, the connector uses 5 threads.

Reviewer: ```suggestion
The number of threads the connector will run in parallel when fetching documents from the SharePoint server. By default, the connector uses 5 threads.
```

```yaml
sharepoint_sync_thread_count: 5
```

#### `enterprise_search_sync_thread_count`

> The number of threads the connector will run parallelly for indexing documents to the Enterprise Search instance. By default, the connector uses 5 threads.

Reviewer: ```suggestion
The number of threads the connector will run in parallel when indexing documents to the Enterprise Search instance. By default, the connector uses 5 threads.
```

```yaml
enterprise_search_sync_thread_count: 5
```

> For the Linux distribution with atleast 2 GB RAM and 4 vCPUs, you can increase the thread counts if the overall CPU and RAM are under utilized i.e. below 60-70%.

Reviewer: ```suggestion
For a Linux distribution with at least 2 GB RAM and 4 vCPUs, you can increase thread counts— if the overall CPU and RAM are underutilized i.e. below 60-70%.
```
